test/AWS.Deploy.CLI.IntegrationTests/WebAppNoDockerFileTests.cs

```diff
@@ -164,7 +164,29 @@ public async Task BeanstalkArmDeployment()
 
             // Verify stack exists in list of deployments
             var listStdOut = _interactiveService.StdOutReader.ReadAllLines().Select(x => x.Split()[0]).ToList();
-            Assert.Contains(listStdOut, (deployment) => _stackName.Equals(deployment));
+            Assert.Contains(listStdOut, (deployment) => _stackName.Equals(deployment));

```

Comment:
do we want/need a test for the opposite of this - arm to x86?

Reply:
I decided against it since all our deployments default to X86 and didn't want to add more deployments/redeployments to further slow down our already slow test suite. The logic is the same for both x86 and arm so I didn't think it would be a must as i am already testing the code path. If you feel strongly about it, i could add that test.